includes/Modules/Tag_Manager.php

```diff
@@ -643,6 +644,20 @@ private function can_analytics_use_snippet( $original_value ) {
 		return $original_value;
 	}
 
+	/**
+	 * Handles analytics tracking opt-out.
+	 *
+	 * @since n.e.x.t
+	 */
+	private function analytics_tracking_opt_out() {
+		$settings       = $this->get_settings()->get();
+		$ga_property_id = $settings['gaPropertyID'];
+		?>
```

Comment:
Here we should return early if no `gaPropertyID` is set, but also if the given `$propertyID` (from param) matches the `gaPropertyID` to not add a duplicate opt-out script.